bower.json

```diff
@@ -1,21 +0,0 @@
-{
-  "name": "rspec-website",
-  "version": "0.0.0",
-  "homepage": "https://github.com/rspec/rspec-website",
-  "authors": [
-    "Andrew <andrew@mootpointer.com>"
-  ],
-  "license": "MIT",
-  "private": true,
-  "ignore": [
-    "**/.*",
-    "node_modules",
-    "bower_components",
-    "vendor/javascripts",
-    "test",
-    "tests"
-  ],
-  "dependencies": {
-    "normalize-css": "~2.1.3"
```

Comment:
I think this is still used no?

Reply:
We do, inline:
```
source/stylesheets/_application.css.scss
1://= normalize
3:@import "normalize";

source/stylesheets/_normalize.css
1:/*! normalize.css v2.0.1 | MIT License | git.io/normalize */
```

It seems we're far behind [the current v8.0.1](https://necolas.github.io/normalize.css/) :D